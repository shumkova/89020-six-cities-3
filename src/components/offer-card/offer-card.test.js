import React from "react";
import renderer from "react-test-renderer";
import OfferCard from "./offer-card";

const offer = {
  img: `img/apartment-01.jpg`,
  title: `Beautiful & luxurious apartment at great location`,
  price: 120,
  type: `Appartment`,
};

const onCardHover = () => {};
const onHeaderClick = () => {};

it(`Render OfferCard`, () => {
  const tree = renderer.create(
      <OfferCard
        offer={offer}
        onCardHover={onCardHover}
        onHeaderClick={onHeaderClick}
      />
  ).toJSON;

  expect(tree).toMatchSnapshot();
});
