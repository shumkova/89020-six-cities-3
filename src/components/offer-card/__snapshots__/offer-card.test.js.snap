// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render OfferCard 1`] = `[Function]`;
